### PR TITLE
Adjust podlet templating

### PR DESCRIPTION
**Pull request by digitalsadhu**
Adjust templating to match changes in utils and layout.

TODO:

- [x] Code changes
- [x] Update/add Tests
- [x] Update utils package once next version is published
- [x] Update proxy package once next version is published
- [x] Update documentation

### CODE REVIEW

File: lib/podlet.js [hunk 1]
```diff
@@ -288,49 +290,61 @@ const PodiumPodlet = class PodiumPodlet {
     }
 
     async process(incoming) {
-        incoming.view = this._view;
-        incoming.name = this.name;
-        incoming.css = this.cssRoute;
-        incoming.js = this.jsRoute;
+        const s = incoming;
```
Comment (trygve-lie) [inline lib/podlet.js]: I am struggeling to understand the no re-assignment ESLint rule here. Having to assign the object to a new variable to alter the objects propertis have as I see it no value. Should we ignore this rule?

File: lib/podlet.js [hunk 2]
```diff
@@ -288,49 +290,61 @@ const PodiumPodlet = class PodiumPodlet {
...
+        return incoming.render(data);
+    }
+
     middleware() {
         return (req, res, next) => {
             const incoming = new HttpIncoming(req, res);
 
             this.process(incoming)
-                .then(async (incom) => {
+                .then(async incom => {
```
Comment (trygve-lie) [inline lib/podlet.js]: We can turn this promise chain into plain `async`/`await`.

File: lib/podlet.js (hunk 1 above)
Comment (digitalsadhu) [reply lib/podlet.js]: yeah, agreed

Comment (digitalsadhu) [reply lib/podlet.js]: done

File: lib/podlet.js (hunk 2 above)
Comment (digitalsadhu) [reply lib/podlet.js]: done

**Pull request merged**